src/clickableqlabel.h

```diff
@@ -0,0 +1,22 @@
+#ifndef CLICKABLELABEL_H
+#define CLICKABLELABEL_H
+
+#include <QLabel>
+#include <QWidget>
+#include <Qt>
+
+class ClickableQLabel : public QLabel { 
+    Q_OBJECT 
+
+public:
+    explicit ClickableQLabel(QWidget* parent = Q_NULLPTR);
+    ~ClickableQLabel();
+
+signals:
+    void clicked();
+
+protected:
+    void mouseReleaseEvent(QMouseEvent* event);
+};
```

Comment:
I start wondering if this class is now justified. Can't you use some kind of a Qt button instead?

Reply:
You are right. Initial target I had was different, so we can change this.